lib/TH/generic/simd/simd.h

```diff
@@ -9,7 +9,7 @@
 #endif
 
 // Can be found on Intel ISA Reference for CPUID
-#define CPUID_AVX2_BIT 0x10       // Bit 5 of EBX for EAX=0x7
+#define CPUID_AVX2_BIT 0x20       // Bit 5 of EBX for EAX=0x7
```

Comment:
Are you sure about that? `0x20` is 6th bit, which is:

1. out of sync from the comment
2. incorrect according to [CPUID page on Wikipedia](https://en.wikipedia.org/wiki/CPUID)